src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml

```diff
@@ -259,6 +247,94 @@ slots:
     range: JgiMtInterface
     inlined: true
     inlined_as_list: true
+  dna_absorb1
+  dna_absorb2
+  dna_concentration
+  cont_type:
+    description: Tube or plate (96-well)
+    title: JGI container type
+    examples:
+      - value: plate
+      - value: tube
+    rank: 10
+    range: JgiContTypeEnum
+    recommended: true
+  cont_well:
+    title: JGI plate position
+    comments:
+      - Required when 'plate' is selected for container type.
+      - Leave blank if the sample will be shipped in a tube.
+      - JGI will not process samples in corner wells, so A1, A12, H1 and H12 will not pass validation.
+      - Fill plate in column-wise format. B1-G1,A2-H2,A3-D3 (NOT A2-A11,B1-B8).
+    examples:
+      - value: B2
+    rank: 11
+    string_serialization: '{96 well plate pos}'
+    recommended: true
+    pattern: ^(?!A1$|A12$|H1$|H12$)(([A-H][1-9])|([A-H]1[0-2]))$
+  container_id:
+    title: Container label
+    comments:
+      - Must be unique across all tubes and plates, and <20 characters. 
+      - All samples in a plate should have the same plate label.
+    examples:
+      - value: Pond_MT_041618
+    rank: 9
+    string_serialization: '{text < 20 characters}'
+    recommended: true
+  dna_dnase:
+    title: DNase treatment
+    comments:
+      - Note DNase treatment is required for all RNA samples.
+    examples:
+      - value: 'no'
+    rank: 13
+    range: YesNoEnum
+    required: true
+  project_contact:
+    title: Sequencing project contact
+    comments:
+      - Do not edit these values. A template will be provided by NMDC in which these
+        values have been pre-filled.
+    examples:
+      - value: John Jones
+    rank: 18
+    string_serialization: '{text}'
+    recommended: true
+  jgi_samp_id:
+    title: JGI Sample ID
+    comments:
+      - Do not edit these values. A template will be provided by NMDC in which these
+        values have been pre-filled.
+    examples:
+      - value: '187654'
+    rank: 3
+    string_serialization: '{text}'
+    slot_group: JGI-Metagenomics
+    recommended: true
+  jgi_sample_format:
+    description: Solution in which the JGI sample has been suspended
+    title: JGI sample format
+    examples:
+      - value: Water
+    rank: 12
+    range: SampleFormatEnum
+    recommended: true
+  jgi_sample_name:
+    description: Give the JGI sample a name that is meaningful to you. Sample names
+      must be unique across all JGI projects and contain a-z, A-Z, 0-9, - and _ only.
+    title: JGI sample name
+    examples:
+      - value: JGI_pond_041618
+    rank: 4
+    string_serialization: '{text}'
+    slot_group: JGI-Metagenomics
```

Comment:
@pkalita-lbl I don't need the slot_group right?

Reply:
so, if it's JGI-Metagenomics and JGI-Metatranscriptomics, I list them both and separate by |? 
I see none of the MIxS slots have this.. so also just for my knowledge, what does this metaslot provide? That we don't get by assigning slots to the classes?